platform/api/equinixmetal/api.go

```diff
@@ -362,6 +362,7 @@ ExecStart=/usr/bin/curl --retry-delay 1 --retry 120 --retry-connrefused --retry-
 # We don't verify signatures because the iPXE script isn't verified either
 # (and, in fact, is transferred over HTTP)
 
+ExecStartPre=-/bin/bash -c 'shopt -s nullglob; for disk in /dev/*da? /dev/nvme?n1; do echo wipefs -af ${disk}; done'
```

Comment:
Maybe, I'm missing something - but:
```suggestion
ExecStartPre=-/bin/bash -c 'shopt -s nullglob; for disk in /dev/*da? /dev/nvme?n1; do wipefs -af ${disk}; done'
```

Reply:
Yes I thought that was the main reason haha.